src/components/Page/Page.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Title from "../Title/Title";
+import CardContainer from '../CardContainer/CardContainer';
+
+class Page extends React.Component {
+    render() {
+        return (
+            <div>
```

Comment:
Are we wrapping the components in a div so that we aren't passing down sibling elements? If so, we can use React Fragments, and then the unnecessary div isn't rendered. Ignore this comment if you **do** actually need that div though

Reply:
ooooh I didn't know there were short tags